scripts/annotate/check_annotations.py

```diff
@@ -30,7 +30,9 @@
 parser.add_argument(
     "-o", "--output",
     help="A copy of the annotation.csv file including the detected problems")
-
+parser.add_argument("--skip-ok", action="store_true", default=False,
```

Comment:
Another wording this could be `--only-errors`. Probably fine to merge as it is and come back to it as it gets tested by more consumers of the utility.